pkgs/development/libraries/nng/default.nix

```diff
@@ -2,13 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "nng";
-  version = "1.5.2";
+  version = "1.6.0-prerelease";
```

Comment:
I'm not too keen on bumping this to a prerelease, if it's necessary specifically for the R package then maybe just override nng there until 1.6.0 is released.

Reply:
Sure, understood. I added nng just for nanonext, which wasn't building successfully otherwise. I'll ask the nanonext author why this dev version was introduced

Reply:
I asked the author and nanonext uses some recent features in nng's dev branch. Overall the dev branch is stable. Since nix's nng derivation is really just for nanonext at this point, I'd like to merge if you're okay with it. Nanonext won't build in nix otherwise. If anyone complains about the system dependency, I can role back and remove the R package for now.